phpphp/homework4.php

```diff
@@ -0,0 +1,12 @@
+<?php
+function checkPrimeNumber2 ($n) {    
+    for ($i = 0; $i <= $n; ++$i) {
+      $a = checkPrimeNumber($i);
+      if ($a == 1) {
+          echo $i;
+          echo "<br>";  
+      }    
+    }  
+}
```

Comment:
indent sometimes 2 spaces, sometimes 1 tab (4 spaces)
can you fix only 1 format?